cdap-ui/app/directives/widget-container/widget-multi-select-dropdown/widget-multi-select-dropdown.html

```diff
@@ -0,0 +1,27 @@
+<!--
+  Copyright © 2017 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="my-multi-select-dropdown">
+  <div
+    ng-dropdown-multiselect=""
```

Comment:
is this needed?

Reply:
That is the angular directive that renders multi-select? http://dotansimha.github.io/angularjs-dropdown-multiselect/docs/#/main I just went with the way they had in their docs or in our dashboard view. Not sure how much of difference it would be with `ng-dropdown-multiselect` vs `ng-dropdown-multiselect=""`. Any specific reason you wanted it as `ng-dropdown-multiselect`?

Reply:
oh I wasn't aware this was a custom directive module, I thought it was built in angular. No reason in particular. 👍